book/accessing_argv.md

```diff
@@ -0,0 +1,98 @@
+# Accessing ARGV
+
+If you have positional and REST parameters along with a flag, you can access them in your main script file in the `main` function - unlike other scripting languages, Nushell doesn't provide direct access to the ARGV input array, but here is a way to circumvent the issue:
+
+```nu
+# supercar.nu
+
+def main [
+  manuf: string
+  model: string
+  --fast (-f)
+  ] {
+    [$manuf $model $fast] | flatten
+}
+```
+
+```nu
+nu supercar.nu manuf ferrari model F40 -f
```

Comment:
I'm not following this example, since I cannot get it to work with the main above. Am I missunderstanding something?

Reply:
No, it seems I screwed up and uploaded a draft instead of the final files!